go.mod

```diff
@@ -200,7 +200,7 @@ require (
 	github.com/gosimple/unidecode v1.0.1 // indirect
 	github.com/gregdel/pushover v1.2.1 // indirect
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
-	github.com/grpc-ecosystem/grpc-gateway/v2 v2.11.3 // indirect
+	github.com/grpc-ecosystem/grpc-gateway/v2 v2.16.0 // indirect
```

Comment:
Read the release notes, didn't see anything concerning.

Reply:
overachiever!

Reply:
I felt I needed to contribute something to this PR, just to feel involved 😅